src/actions/articleAction/actionTypes.js

```diff
@@ -1,3 +1,6 @@
 export const FETCH_ARTICLE_START = 'FETCH_ARTICLE_START';
 export const FETCH_ARTICLE_SUCCESS = 'FETCH_ARTICLE_SUCCESS';
 export const FETCH_ARTICLE_FAILED = 'FETCH_ARTICLE_FAILED';
+
+export const FETCH_FOLLOWERS_START = 'FETCH_FOLLOWERS_START';
+export const FETCH_FOLLOWERS_COMPLETE = 'FETCH_FOLLOWERS_COMPLETE';
```

Comment:
Hi @Danielshow. Why are you adding this in the articleActions file. You could add this to your own actionType file

Reply:
Because I triggered the action in article view actions, that is where it is supposed to be.